log/keyvalue_test.go

```diff
@@ -70,6 +70,31 @@ func TestValueEqual(t *testing.T) {
 	}
 }
 
+func TestSortedValueEqual(t *testing.T) {
+	testCases := []struct {
+		value  log.Value
+		value2 log.Value
+	}{
+		{
+			value: log.MapValue(
+				log.Slice("l", log.IntValue(3), log.StringValue("foo")),
+				log.Bytes("b", []byte{3, 5, 7}),
+				log.Empty("e"),
+			),
+			value2: log.MapValue(
+				log.Bytes("b", []byte{3, 5, 7}),
+				log.Slice("l", log.IntValue(3), log.StringValue("foo")),
+				log.Empty("e"),
+			),
+		},
+	}
+	for _, tc := range testCases {
+		t.Run(tc.value.String(), func(t *testing.T) {
+			assert.Equal(t, true, tc.value.Equal(tc.value2), "%v.Equal(%v)", tc.value, tc.value2)
+		})
+	}
```

Comment:
This is a lot of boilerplate for a single test.

```suggestion
	b := log.Bytes("b", []byte{3, 5, 7})
	s := log.Slice("l", log.IntValue(3), log.StringValue("foo"))
	e := log.Empty("e")
	m0, m1 := log.MapValue(s, b, e), log.MapValue(b, s, e)
	assert.True(t, m0.Equal(m1), "%v.Equal(%v)", m0, m1)
```

Reply:
@dmathieu thoughts?

Reply:
Ah yes, sorry, I missed that notification :(
I agree. I setup this test when there were both slices and maps. But without slices, we can make it simpler.